README.md

```diff
@@ -68,7 +68,7 @@ Please visit [Roadmap](docs/roadmap.md) if you are interested in Amundsen upcomi
 
 ## Community
 
-Want help? Join our [slack channel](https://bit.ly/2tFsN1X)
+Want help? Join our [slack channel](https://bit.ly/2tFsN1X). For other discussion, please use the [Amundsen mailing list](amundsen-dev@googlegroups.com).
```

Comment:
We need to use email markup for this or else it treats it as normal link and goes to a 404 page. 

Also does this alias with `@googlegroups.com` work, can you send a test email?  To my knowledge I thought all we had setup was `amundsen-dev@lyft.com`.

Reply:
this is not our email alias. I manually created this google group for external people. Currently I just add myself which requires others to join the group by themselves.